apps/api/.env.example

```
@@ -0,0 +1,3 @@
+DATABASE_URL="postgresql://postgres:postgres@localhost:5432/brawltome?schema=public"
+BRAWLHALLA_API_KEY="REPLACE_ME"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)